### DISCUSSION TITLE
Is it possible to do Full Disk Encryption on orange Pi using this distro?

### DISCUSSION THREAD

**Discussion by passcombo** [Joshua-Riek/ubuntu-rockchip#484] [Q&A] [answered]
I've read new Ubuntu supports encryption but requires UEFI which can lead to an error when trying to secure the drive?
Can you provide some steps to do this properly? I'm kind of confused 21 century welcomes us with no easy way to have security onboard by default.

**Comment by Joshua-Riek** [comment, marked as answer]
This project uses U-Boot and not a UEFI because it is not mature enough.

If you are worried about security, I would recommend against RK3588, as the only mature kernel comes from Android and missing so many security patches.
